fighters/fox/src/acmd/specials.rs

```diff
@@ -100,40 +100,74 @@ unsafe fn sound_specialhi(fighter: &mut L2CAgentBase) {
 unsafe fn game_speciallwstart(fighter: &mut L2CAgentBase) {
     let lua_state = fighter.lua_state_agent;
     let boma = fighter.boma();
-    frame(lua_state, 1.0);
+    frame(lua_state, 0.0);
     if is_excute(fighter) {
+        FT_MOTION_RATE(fighter, 2.0);
```

Comment:
once again people are putting `FT_MOTION_RATE` in `is_excute`, don't do that

Reply:
oops, why?